src/components/Library.js

```diff
@@ -1,30 +1,25 @@
-import React from "react";
+import React, { useCallback } from "react";
 import LibrarySong from "./LibrarySong";
 
 const Library = ({
   songs,
+  currentSongId,
```

Comment:
Tutaj teoretycznie można byłoby spokojnie przekazać cały `currentSong`, ale jeśli jest prosta opcja, to lepiej przekazać do komponentu tylko te informacje, które w rzeczywistości potrzebuje.